src/docfx/docfx.csproj

```diff
@@ -14,7 +14,7 @@
   <ItemGroup>
     <PackageReference Include="Azure.Identity" Version="1.2.0-preview.1" />
     <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.0.3" />
-    <PackageReference Include="docs.validation" Version="1.0.0-beta-00129-f257836" />
+    <PackageReference Include="docs.validation" Version="1.0.0-beta-00141-21b1204" />
```

Comment:
@OsmondJiang I noticed that we are referencing packages from the Docs.Validation `develop` branch and it is ahead of `master`, even after release. Is there a release process to merge with master or should we consolidate the branches?

Reply:
Docs.Validation is still under fast development process, we currently using the package version released from developer branch directly in docfx v3 which is fast and convenient, later once we are stable, we will use master to control the release process, good point!